src/operator/custom/custom.cc

```diff
@@ -181,8 +181,22 @@ Operator* CustomOpProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_sh
 }
 
 MXNET_REGISTER_OP_PROPERTY(Custom, CustomOpProp)
-.describe("Custom operator implemented in frontend.")
-.add_argument("op_type", "string", "Type of custom operator. Must be registered first.");
+.describe(R"code(Apply a custom operator implemented in the frontend language (Python).
+
+Custom operators can be created in Python by subclassing `mx.operator.
```

Comment:
we should make the cc doc general, instead of python specific. and explain that an operator should have forward, backward, infer_shape blarbar

we can move the python examples into symbol_doc.py